sal/class.json-api-site-jetpack.php

```diff
@@ -138,8 +138,49 @@ function allowed_file_types() {
 		return $allowed_file_types;
 	}
 
+	/**
+	 * Return site's privacy status.
+	 *
+	 * @return boolean  Is site private?
+	 */
 	function is_private() {
-		return false;
+		return (int) $this->get_atomic_cloud_site_option( 'blog_public' ) === -1;
+	}
+
+	/**
+	 * Return site's coming soon status.
+	 *
+	 * @return boolean  Is site "Coming soon"?
+	 */
+	function is_coming_soon() {
+		return $this->is_private() && (int) $this->get_atomic_cloud_site_option( 'wpcom_coming_soon' ) === 1;
+	}
+	
+	/**
+	 * Return site's launch status.
+	 *
+	 * @return string|boolean  Launch status ('launched', 'unlaunched', or false).
+	 */
+	function get_launch_status() {
+		return $this->get_cloud_site_option( 'launch-status' );
```

Comment:
`get_cloud_site_option` does not exist in Jetpack though, so with this patch we end up with a Fatal when we query the sites endpoint:

```
Uncaught Error: Call to undefined method Jetpack_Site::get_cloud_site_option() in /wp-content/plugins/jetpack-dev/sal/class.json-api-site-jetpack.php:165
```

Reply:
Good eye! It got in there when merging the other PR into this one. It's fixed now